workflow/json/fix.go

```diff
@@ -0,0 +1,12 @@
+package json
+
+import "strings"
+
+func FixJSONString(s string) string {
+	// I've never encountered something so fundamentally broken in any programming language as the spacing of HTML characters in Golang's JSON parsing.
+	// Any code that utilize will JSON in Golang will always have edge-case bugs related to >, < and &
```

Comment:
Have you tried something like the following?
```
package main

import (
	"encoding/json"
	"log"
	"bytes"
	"fmt"
)

func main() {
	buf := new(bytes.Buffer)
	enc := json.NewEncoder(buf)
	enc.SetEscapeHTML(false)

	v := make(map[string]string)
	v["key"] = "<, >, &"
	if err := enc.Encode(&v); err != nil {
		log.Println(err)
	}

	fmt.Printf("Result: %v", buf.String())
}
```
Perhaps wrapping above into a custom `JSONMarshal()` might be a more generic solution.

Reply:
Typo in comment:
>  utilize will JSON in Golang

Beside that, after reading the comment I'm still not sure what the problem is it tries to solve.